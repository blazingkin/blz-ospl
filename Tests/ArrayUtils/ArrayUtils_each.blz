:main
    arr = ["apple", "peach", "cherry"]
    print("Test 00 each print strings")
    arr.each(print)
    print("----------------------------")
end