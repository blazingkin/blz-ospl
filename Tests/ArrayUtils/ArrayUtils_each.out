Test 00 each print strings
apple
peach
cherry
----------------------------
